docker-compose-pool.yaml

```diff
@@ -63,8 +63,11 @@ services:
     expose:
       # ZMQ socket port.
       - 5555
+      # sleep is required to wait WPE until KME starts and register worker.
+      # This is temporary work around.
     command: |
       bash -c "
+        sleep 5s
```

Comment:
This should be handled by WPE Enclave Manager

Reply:
yes